source/Octopus.Tentacle/Kubernetes/Synchronisation/KubernetesModule.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Autofac;
+using Octopus.Tentacle.Kubernetes.Synchronisation.Internal;
+
+namespace Octopus.Tentacle.Kubernetes.Synchronisation
+{
+    public class SynchronisationModule : Module
```

Comment:
I don't mind a separate Autofac module but I think if you make the `SemaphoreSlimReleaser` and `SemaphoreSlimWithReferenceCount` classes private, you would only need to register the `ReferenceCountingKeyedLock` so probably not worth having a separate Module.